### PR TITLE
Admin UI talks to the API/DB

### PR DESCRIPTION
**Pull request by jasalisbury**
**Description of change**

 * Admin page now saves user changes up to the database
 * Add user seeder
 * add test users as a seeder
 * created at and updated at for users and permissions have default values of "now"
 * users "title" column renamed to "role"
 * GET `/user` returns the currently logged in user from the DB

**Notes**

 * Would be nice to have a validation on permissions that asserts any "global" scope id (the admin scope, "2") has to have a region id of 14 (no region). I tried to get a [model validation](https://sequelize.org/v5/manual/models-definition.html#model-wide-validations) working but something wasn't working correctly. Either need a ticket to write the validation or need to pair with someone to figure it out.

**How to test**

1) Go to https://tta-smarthub-dev.app.cloud.gov/admin
2) Select a user, update a field and save
3) Reload the page and note the updated field has the proper value

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/52

**Checklist**

- [x] Meets issue criteria
- [x] Code tested
- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [x] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

File: frontend/src/fetchers/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+export const get = async (url) => {
+  const res = await fetch(url, {
```
Comment (rahearn) [inline frontend/src/fetchers/index.js:2]: This should be using axios for IE11, right?

**Review by rahearn — CHANGES_REQUESTED** [review]
I'm concerned at this point that we're still hardcoding the session user id instead of utilizing the `findOrCreateUser` function from `accessValidation.js`. If that isn't already scheduled work I think we need to just roll it into this PR.

File: frontend/src/fetchers/index.js (hunk 1 above)
Comment (jasalisbury) [reply frontend/src/fetchers/index.js:2]: IE11 support for fetch (among other things) is handled by the [polyfill](https://github.com/HHS/Head-Start-TTADP/blob/ff221a79a71ad1f1a09f8bac1f145088db77d8e3/frontend/src/index.js#L1) (https://www.npmjs.com/package/react-app-polyfill)

**Comment by jasalisbury** [comment]
> I'm concerned at this point that we're still hardcoding the session user id instead of utilizing the `findOrCreateUser` function from `accessValidation.js`. If that isn't already scheduled work I think we need to just roll it into this PR.

One consequence of this change that I just realized... We will need to come up with some way of setting which users are Admins. Two ways I can think of off the top of my head:
1) We get a list of the `hsesUserId` `email` pairs and write a seed/migration adding the users and permissions
2) We write a stand alone script (run from the cli) that can elevate a normal user to Admin

1 seems like a good solution for prod/staging where hses ids should stay static. Not sure what happens to hses ids in the test environment when hses resets all accounts.

**Comment by rahearn** [comment]
> 2\. We write a stand alone script (run from the cli) that can elevate a normal user to Admin

I think I prefer this one because it will work no matter the environment. Something that can be run with `cf run-task`. Can you create a card for that?

**Comment by jasalisbury** [comment]
> > 2. We write a stand alone script (run from the cli) that can elevate a normal user to Admin
> 
> I think I prefer this one because it will work no matter the environment. Something that can be run with `cf run-task`. Can you create a card for that?

https://github.com/HHS/Head-Start-TTADP/issues/243

**Review by rahearn — APPROVED** [review]

**Pull request merged**